app/models/concerns/pay/subscription_extensions.rb

```diff
@@ -5,6 +5,7 @@ module SubscriptionExtensions
     included do
       after_commit :send_admin_notification
       after_commit :send_subscribed
+      after_commit :send_cancel_subscription if Feature.enabled?(:cancel_subscription)
```

Comment:
This syntax looks weird. Shouldn't it be this?

```suggestion
      after_commit :send_cancel_subscription, if: Feature.enabled?(:cancel_subscription)
```

Reply:
Earlier code worked while testing but yeah it does not follow correct syntax. I have updated to use below syntax as the one mentioned in suggestion was throwing error. Have tested code locally.
     
after_commit :send_cancel_subscription, if: -> { Feature.enabled?(:cancel_subscription) }

Referred syntax from this article - https://engineering.gusto.com/the-rails-callbacks-best-practices-used-at-gusto/